### DISCUSSION TITLE
Connect SD Card directly to ESP3D 3.0

### DISCUSSION THREAD

**Discussion by Gokuh1980** [luc-github/ESP3D#720] [Questions]
Hi! it´s possible to attach a sd card module to a wemos d1 mini running ESP3D 3.0 and run gcodes from there? i´ve been reading a lot and couldn´t find precise information about this

I really like this system (more than octoprint) but this sd card file low speed transfer to printer really makes me looko in other ways... Already bought an EZSHARE only to find out that only transfer multimedia files, and even worse Ender 3V2 doesnt recognize the micro sd...

Any help would be appreciate. Thanks

**Comment by luc-github** [comment]
> it´s possible to attach a sd card module to a wemos d1 mini running ESP3D 3.0  

 Yes
 
 >run gcodes from there   

 Not yet 
 
 Mitigation:  https://github.com/luc-github/ESP3D/discussions/535
